network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/image_feature_block.html

```diff
@@ -0,0 +1,27 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+<div class="container">
+<div class="row">
+
+  <div class="d-xl-block feature mb-md-5">
+    <div class="position-relative h-100 d-xl-flex justify-content-end">
+      <div class="feature-image">
+        {% image value.image fill-928x400 as img %}
+        <img src="{{ img.url }}" alt="{{ value.altText }}">
+      </div>
+      <div class="feature-content align-self-center">
+        <div class="tw-h6-heading d-md-block mb-2">{{ value.metadata }}</div>
```

Comment:
What do you think about wrapping this in an `{% if %}` block, since the `metadata` is not required to be added by the editors?